package.json

```diff
@@ -68,67 +71,70 @@
   },
   "dependencies": {
     "async": "^2.5.0",
-    "autoprefixer": "^8.5.2",
+    "autoprefixer": "^9.5.1",
     "axe-core": "3.0.3",
-    "babel-loader": "^7.1.2",
+    "babel-loader": "^8.0.5",
     "babel-polyfill": "^6.23.0",
-    "browserslist-config-terra": "^1.0.0",
     "chai": "^4.2.0",
     "chalk": "^2.4.2",
-    "clean-webpack-plugin": "^0.1.17",
+    "clean-webpack-plugin": "^2.0.1",
     "clone": "^2.1.1",
     "commander": "^2.19.0",
-    "css-loader": "^0.28.7",
+    "css-loader": "^2.1.1",
+    "cssnano": "^4.1.10",
+    "cssnano-preset-advanced": "^4.0.7",
     "duplicate-package-checker-webpack-plugin": "^3.0.0",
     "express": "^4.16.4",
-    "file-loader": "^1.1.5",
-    "fs-extra": "^6.0.1",
+    "file-loader": "^3.0.1",
+    "fs-extra": "^7.0.1",
     "glob": "^7.1.1",
     "ip": "^1.1.5",
     "load-json-file": "^5.0.0",
     "memory-fs": "^0.4.1",
     "mime-types": "^2.1.21",
-    "mini-css-extract-plugin": "^0.4.5",
+    "mini-css-extract-plugin": "^0.6.0",
     "node-sass": "^4.11.0",
-    "postcss": "^6.0.9",
-    "postcss-assets-webpack-plugin": "^2.0.0",
-    "postcss-custom-properties": "^7.0.0",
-    "postcss-loader": "^2.0.6",
+    "postcss": "^7.0.16",
+    "postcss-assets-webpack-plugin": "^3.0.0",
+    "postcss-custom-properties": "^8.0.10",
+    "postcss-loader": "^3.0.0",
     "postcss-rtl": "^1.3.3",
     "raf": "^3.4.1",
-    "raw-loader": "^0.5.1",
+    "raw-loader": "^2.0.0",
     "sass-loader": "^7.0.1",
     "shelljs": "^0.8.3",
     "terra-aggregate-translations": "^1.0.0",
     "terser-webpack-plugin": "^1.1.0",
     "wdio-visual-regression-service": "^0.9.0",
     "webdriverio": "^4.14.1",
```

Comment:
Are there plans as part of terra-toolkit v5 update to update to webdriver v5?

Reply:
Webdriver v5 isn't stable yet IMO. There are issues and PR moving in/out of that repo daily

Reply:
I also haven't tried it yet